libs/input-mapping/src/Helper/RealDevStorageTagsRewriteHelper.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\InputMapping\Helper;
+
+use Keboola\InputMapping\Exception\InvalidInputException;
+use Keboola\InputMapping\File\Options\InputFileOptions;
+use Keboola\InputMapping\File\Options\RewrittenInputFileOptions;
+use Keboola\StorageApi\Options\ListFilesOptions;
+use Keboola\StorageApiBranch\ClientWrapper;
+use Psr\Log\LoggerInterface;
+
+class RealDevStorageTagsRewriteHelper implements TagsRewriteHelperInterface
+{
+    public const MATCH_TYPE_EXCLUDE = 'exclude';
+    public const MATCH_TYPE_INCLUDE = 'include';
+
+    public function rewriteFileTags(
+        InputFileOptions $fileConfigurationOriginal,
+        ClientWrapper $clientWrapper,
+        LoggerInterface $logger
+    ): RewrittenInputFileOptions {
```

Comment:
Pokusil jsem se zachovat stejnou strukturu jako u FakeDevTagsRewiteHelper, i když je dementní, ale aby bylo vidět, že to dělá zhruba to stejný